tests/scene/test_path_follow_3d.h

```diff
@@ -283,4 +283,57 @@ TEST_CASE("[SceneTree][PathFollow3D] Calculate forward vector") {
 
 	memdelete(path);
 }
+
+TEST_CASE("[SceneTree][PathFollow3D] Calculate forward vector with degenerate curves") {
+	Ref<Curve3D> curve = memnew(Curve3D);
+
+	// Form a circle using 2 cubic and 2 quadratic curves
```

Comment:
All 4 sub-curves are cubic (besides not really a circle).
```suggestion
```

If anything, I'd explain what's degenerate about this curve (equal control points).